addons/mail/static/src/messaging/component/thread_textual_typing_status/thread_textual_typing_status.js

```diff
@@ -0,0 +1,49 @@
+odoo.define('mail.messaging.component.ThreadTextualTypingStatus', function (require) {
+'use strict';
+
+const components = {
+    ThreadTypingIcon: require('mail.messaging.component.ThreadTypingIcon'),
+};
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component } = owl;
+
+class ThreadTextualTypingStatus extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const thread = this.env.entities.Thread.get(props.threadLocalId);
+            return {
+                thread: thread ? thread.__state : undefined,
+            };
```

Comment:
Will it observe orderedOtherTypingMembers length?

Reply:
`orderedOtherTypingMembers` is a x2many relational field, so this is a list of local ids in store, so it will be observed in `__state`.

Reply:
ok